### PR TITLE
fix layout

### PR DESCRIPTION
**Pull request by tuannd255**

### CODE REVIEW

File: app/views/courses/show.html.erb [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
-          <span><%= user_task.task.name %></span>
-          <% unless user_task.status %>
-            <%= form_for [@course, user_task] do |f|%>
-              <span>
-                <%= f.hidden_field :status, value: true %>
-                <%= f.submit t("admin.finish"), 
-                  class: "btn btn-primary hl-button" %>
-              </span>
+<div class="row">
+<div class="col-md-6">
```
Comment (VuThiHien94) [inline app/views/courses/show.html.erb]: ident

File: app/controllers/supervisor/courses_controller.rb [hunk 2]
```diff
@@ -2,11 +2,12 @@ class Supervisor::CoursesController < ApplicationController
...
 
   def index
-    @courses = current_user.courses.page params[:page]
+    @courses = current_user.courses.paginate page: params[:page],
+      per_page: Settings.page
   end
 
   def show
-    @users_in_course = @course.user_courses.includes :users
+    @users_in_course = @course.user_courses.includes :user
```
Comment (tiennh1995) [inline app/controllers/supervisor/courses_controller.rb]: @users_in_course = @course.user_courses.includes(:user, user_subjects: [:subject,  user_tasks: [:task] ])

File: app/controllers/admin/users_controller.rb [hunk 3]
```diff
@@ -3,7 +3,7 @@ class Admin::UsersController < ApplicationController
   before_action :get_role, only: [:new, :edit]
 
   def index
-  end
+    @users = @users.page params[:page]
```
Comment (tiennh1995) [inline app/controllers/admin/users_controller.rb]: Thiếu and

File: app/controllers/courses_controller.rb [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
   load_and_authorize_resource only: :show
-  
+
   def index
     @courses = current_user.courses.page params[:page]
   end
 
   def show
     @user_course = @course.user_courses.find_by user_id: current_user.id
+    @user_in_course = @course.users
```
Comment (tiennh1995) [inline app/controllers/courses_controller.rb]: @users_in_course = @course.user_courses.includes(:user, user_subjects: [:subject, user_tasks: [:task] ])

File: app/views/courses/show.html.erb [hunk 5]
```diff
@@ -1,26 +1,39 @@
...
+    <ul>
+      <% @user_course.user_subjects.each do |user_subject| %>
+        <h3><li><%= user_subject.subject.name %></li></h3>
+        <% user_subject.user_tasks.each do |user_task|%>
```
Comment (duccm94) [inline app/views/courses/show.html.erb]: space trước %>

File: app/views/courses/show.html.erb [hunk 6]
```diff
@@ -1,26 +1,39 @@
...
+          <div>
+            <span><%= user_task.task.name %></span>
+            <% unless user_task.status %>
+              <%= form_for [@course, user_task] do |f|%>
```
Comment (duccm94) [inline app/views/courses/show.html.erb]: space trước %>

File: app/views/static_pages/_admin_home.html.erb [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+        </div>
+      </div>
+    </div>
+    <a href="#">
+    <div class="panel-footer">
+      <span class="pull-left"><%= link_to t("home.view_detail"),
+        admin_users_path %></span>
+      <span class="pull-right"><i class="fa fa-arrow-circle-right"></i></span>
+      <div class="clearfix"></div>
+    </div>
```
Comment (duccm94) [inline app/views/static_pages/_admin_home.html.erb]: 15-20 indent

File: config/locales/en.yml [hunk 8]
```diff
@@ -83,14 +85,15 @@ en:
...
     content: Framgia Training System
     title: Home
+    view_detail: View Details
   log_in: Log in
   log_out: Log out
   mail:
     accept_in_course: You had assigned in course %{content}
     blank: ==============================
-    course: "Course: %{content}"
+    course: 'Course: %{content}'
```
Comment (tiennh1995) [inline config/locales/en.yml]: double quote

File: app/views/static_pages/_supervisor_home.html.erb [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+        </div>
+      </div>
+    </div>
+    <a href="#">
+    <div class="panel-footer center">
+      <span class="pull-center"><%= link_to t("home.view_detail"),
+        supervisor_courses_path %></span>
+      <span class="pull-right"><i class="fa fa-arrow-circle-right"></i></span>
+      <div class="clearfix"></div>
+    </div>
```
Comment (duccm94) [inline app/views/static_pages/_supervisor_home.html.erb]: 12-17 indent

File: app/views/static_pages/_trainee_home.html.erb [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+        </div>
+      </div>
+    </div>
+    <a href="#">
+    <div class="panel-footer center">
+      <span class="pull-center"><%= link_to t("home.view_detail"),
+        courses_path %></span>
+      <span class="pull-right"><i class="fa fa-arrow-circle-right"></i></span>
+      <div class="clearfix"></div>
+    </div>
```
Comment (duccm94) [inline app/views/static_pages/_trainee_home.html.erb]: 12-17 indent

File: app/views/static_pages/_admin_home.html.erb [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+    <a href="#">
+      <div class="panel-footer">
+        <span class="pull-left"><%= link_to t("home.view_detail") %></span>
+        <span class="pull-right"><i class="fa fa-arrow-circle-right"></i></span>
+        <div class="clearfix"></div>
+      </div>
+    </a>
+  </div>
+</div>
```
Comment (duccm94) [inline app/views/static_pages/_admin_home.html.erb]: end file

File: config/locales/en.yml [hunk 12]
```diff
@@ -91,7 +94,7 @@ en:
     course: "Course: %{content}"
     delete_from_course: You had been delete in course %{content}
     description: "Description: %{content}"
-    mail_month: mail for course's infomation
+    mail_month: mail for course"s infomation
```
Comment (tiennh1995) [inline config/locales/en.yml]: single quote

**Comment by tiennh1995** [comment]
ok

File: app/views/static_pages/_admin_home.html.erb [hunk 13]
```diff
@@ -0,0 +1,91 @@
...
+        <div class="col-xs-3">
+          <i class="fa fa-tasks fa-5x"></i>
+        </div>
+        <div class="col-xs-9 text-right">
+          <div class="huge"><%= Course.count %></div>
+          <div><%= t "courses.title" %></div>
+        </div>
+      </div>
+    </div>
+    <a href="#">
```
Comment (hellovietnam93) [inline app/views/static_pages/_admin_home.html.erb]: dùng link_to đi em

File: app/controllers/static_pages_controller.rb [hunk 14]
```diff
@@ -1,5 +1,9 @@
 class StaticPagesController < ApplicationController
   def home
+    @user_size = User.all.size
```
Comment (hellovietnam93) [inline app/controllers/static_pages_controller.rb]: sửa dụng metaprogramming nhé

File: app/controllers/static_pages_controller.rb [hunk 15]
```diff
@@ -1,7 +1,17 @@
...
 
   def help
   end
+
+  private
+  def load_data_size
+    objects = ["user", "course", "subject", "activity"]
+    objects.each do |object|
+      size = "#{object.capitalize}".constantize.all.size
+      instance_variable_set("@#{object}_size", size)
```
Comment (hellovietnam93) [inline app/controllers/static_pages_controller.rb]: bỏ ()

**Pull request merged**